backend/tests/unit_tests/config_definition/payloads/payloads.json

```diff
@@ -73,29 +77,87 @@
     "update": {
       "test_update_w_index": {
         "config_key": "sample_config",
-        "indexes": ["name"]
+        "schema": {
+          "json_schema": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "date": {
+                "type": "string"
+              }
+            }
+          },
+          "index": ["date"]
+        },
+        "indexes": ["name"],
+        "return_value": {
+            "rows_affected": 0,
+            "response": [{"indexname": "date"}]
+        }
       },
       "test_update_d_index": {
         "config_key": "sample_config",
+        "schema": {
+          "json_schema": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "date": {
+                "type": "string"
+              }
+            }
+          },
+          "indexes": ["date"]
+        },
```

Comment:
@coderabbitai Thank you for the suggestion.

While it is important to keep the consistency across test cases, it is also important to understand that the `JSON` file here is supposed to be flexible, meaning, the payloads may or may not contain the exact keys/values. We use `.get()` methods in the payload extractor utility, thus it will be received as `None`, which is appropriate if the `return_value` variable is not being used anywhere.